.github/workflows/test.yml

```diff
@@ -18,9 +18,7 @@ jobs:
       uses: ruby/setup-ruby@v1
       with:
         ruby-version: 2.7.1
```

Comment:
From that other PR, if you pull this into a `.ruby-version` this line can also be skipped

Reply:
You're right, merged before I saw. `spec/ci_ruby_version_spec.rb` would also need to be updated. If you'd like to make a PR, go ahead, otherwise I'll get to it, probably another day.

Reply:
Let me know if you guys need me to do anything on my end. I believe it just needs a license so I'm learning as I go. Thank you so much guys!

Reply:
On second thought, I'd rather not use `.ruby_version` because it introduces unnecessary (IMO) friction when developing locally. Not having the precise version installed has never caused me any problems. Limiting it to CI with ruby-version seems like a good compromise.